libs/java/cert_refresher/src/main/java/com/oath/auth/KeyRefresher.java

```diff
@@ -102,6 +107,38 @@ public KeyRefresher(final String athenzPublicCert, final String athenzPrivateKey
         this.keyManagerProxy = keyManagerProxy;
         this.trustManagerProxy = trustManagerProxy;
         this.keyRefresherListener = keyRefresherListener;
+        
+        // Initialize OpenTelemetry metrics if enabled and available
+        this.otelMetric = initOtelMetrics();
+    }
+
+    /**
+     * Initialize OpenTelemetry metrics if explicitly enabled.
+     * 
+     * Behavior:
+     * - If athenz.cert_refresher.otel_disabled=false: Initialize OTel metrics
+     * - If athenz.cert_refresher.otel_disabled=true or not set: OTel disabled, return null
+     * - If OTel JARs are not available on classpath: Log warning and return null
+     * 
+     * @return OpenTelemetryCertReloadEventEmitter instance or null if disabled or unavailable
+     */
+    private OpenTelemetryCertReloadEventEmitter initOtelMetrics() {
+        String otelDisabledProp = System.getProperty(PROP_OTEL_DISABLED);
```

Comment:
this double negative setting doesn't look right. since we want the default to be false and only enable when configured (at list for this release), we should have the setting called

athenz.cert_refresher.otel_enabled and the default value would be false if not configured. then the check is more natural:

boolean otelEnabled = Boolean.parseBoolean(System.getProperty(PROP_OTEL_ENABLED, "false"));